src/bitset.hpp

```diff
@@ -24,9 +24,15 @@
 
 class Bitset {
  public:
+  using BitsetPair = std::pair<Bitset, Bitset>;
+  // enum orientation = { sorted=0, rotated=1 }
+
   explicit Bitset(std::vector<bool> value);
   explicit Bitset(size_t n, bool initial_value = false);
   explicit Bitset(std::string);
+  explicit Bitset(std::string, std::string);
```

Comment:
Given that these aren't obvious can you add some comment strings?

Reply:
Rather than overloading Bitset constructor, made a static Subsplit() "constructor" function.